portfolio/src/main/webapp/index.html

```diff
@@ -72,10 +72,19 @@ <h1 class="title">Ani Machaidze</h1>
             </div> 
             
             <div id="comments">
+                <label for="number" onchange="getComments()">Choose number of comments</label>
+                    <select name="number" id="number">
+                        <option value="5">5</option>
+                        <option value="10">10</option>
+                        <option value="15">15</option>
+                        <option value="20">20</option>
+                    </select>
+                <br>
                 <button onclick="getComments()">See all comments</button>
```

Comment:
I think this no longer gets all of the comments since it checks the value is `#number`, maybe add a separate function for all comments?